src/lib/OLED/devOLED.cpp

```diff
@@ -25,9 +28,9 @@ static int event()
     {
         return DURATION_IGNORE; // we are still displaying the startup message, so don't change the timeout
     }
-    OLED.updateScreen(OLED.getPowerString((PowerLevels_e)POWERMGNT::currPower()),
-                        OLED.getRateString((expresslrs_RFrates_e)ExpressLRS_currAirRate_Modparams->enum_rate),
-                        OLED.getTLMRatioString((expresslrs_tlm_ratio_e)(ExpressLRS_currAirRate_Modparams->TLMinterval)), thisCommit);
+    OLED.updateScreen(OLED.getPowerString((PowerLevels_e)(config.GetPower())),
+                        OLED.getRateString((expresslrs_RFrates_e)(config.GetRate())),
+                        OLED.getTLMRatioString((expresslrs_tlm_ratio_e)(config.GetTlm())), thisCommit);
```

Comment:
Should be casting them to the enums if we're just using them as int's in the functions.